### ISSUE TITLE
How do I fix erro90 message to 

### ISSUE THREAD

**Issue by michael809-byte** [ish-app/ish#1243]

**Comment by 62f** [comment]
I believe that iSH is trying to pass your message as data to a nonexistent device, perhaps USB or network. Otherwise it’s just the Linux kernel doing its job: by not crashing instead.

**Comment by michael809-byte** [comment]
I'm trying to type something for one of my scrips that I got from GitHub but it's telling me that there's to many things that i typed but when I try it in cloud shell it works so any way I can bypass it?

**Comment by michael809-byte** [comment]
Yes I'm trying to pass data to a device but that device exists because I tried doing the same thing in cloud shell and it worked so can I bypass the error in ish?

**Comment by 62f** [comment]
A good place to start making OS comparisons is from the standpoint of having some fs loaded besides the Alpine-MiniFs. A kernel errno is such a generalized response that I don't know of a reliable way to intercept its root cause; without knowing the specific glue for handling your input I can only guess the approximate behaviors associated with 'bypassing failover', which would be that your data: 1) gets sent to /dev/null 2) gets redirected to a file cumulatively, or 3) arrives at the targeted destination, perhaps corrupted.

**Comment by tbodt** [comment]
> When ever I try to type a bunch of things that I need to write something it tells me Errno 90 Message too large and how do I fix this?

Can you provide more details? I can't figure out what you were trying to do or what went wrong.

**Comment by michael809-byte** [comment]
I’m trying to use a command in my script but it gives me a error saying
OSErrno90 Message to large and above that it says things like python
self.run and all that so any way I could fix that?

**Comment by tbodt** [comment]
> I’m trying to use a command in my script but it gives me a error saying OSErrno90 Message to large and above that it says things like python self.run and all that so any way I could fix that?
> […](#)

Unfortunately this is far from enough detail for me to get any kind of picture of what you're doing.
It sounds like there's a python stack trace, could you paste or screenshot that?

**Comment by michael809-byte** [comment]
Sure here’s the screenshot 
![8A3E3BA4-C8F7-453E-A628-AE788F1AAB9D](https://user-images.githubusercontent.com/77212666/105417400-2fb44c80-5c01-11eb-93f1-d51986b8137d.jpeg)

**Comment by tbodt** [comment]
This must mean the message is too large for a UDP packet. How large is the message?

**Comment by michael809-byte** [comment]
40

**Comment by 62f** [comment]
How unfortunate, that method is unsupported.
<details> 
title : 50.34.44.240 PuTTY: Bots 30k
color C
echo off & cls
echo ============================== 
echo  Welcome To The SinFull Botnet 
echo ==============================
set /p root=Username: 
set /p root=Password: 
if "%sinner%"=="pump" if "%pass%"=="security" goto :B
cls
echo Made By I hit you v1, most credit goes to a freind of mines whom taught me python
echo Most/Everything im available to do today is mainly for that freind kicking me into it so big thanks.
echo Bigest attack: 30k gbps                                                                                    $15 For All The Methods     
echo Verison ="3.2"                                                                                             $15 A Month For The Botnet
echo              DDoS METHODS                           TOOLS           BASIC COMMANDS     PAYED METHODS
echo UDP  HOST PORT TIMEOUT SIZE  UDP ATTACK       Stopattacks:          clear:          DROP: UDPMIX: DNS:
echo ICMP HOST PORT TIMEOUT SIZE  ICMP ATTACK      GRAB A DOMIANS IP:    Exit:           INFECT: SMASH: POUND:
echo SYN  HOST PORT TIMEOUT SIZE  SYN ATTACK       PING A HOST:          Methods         LAG: CRASH: RAPE: ATD:
echo HTTP HOST PORT TIMEOUT SIZE  SYN ATTACK       Portscan HOST RANGE:  Tools   
</details>
ping localhost >nul     
set /p IP= SinFull@Botnet:
:top
PING  -n 1 %IP% | FIND "TTL="
IF ERRORLEVEL 1 (SET IN=1F  & echo Attack Sent...offline) ELSE (SET IN=4F)
color C
ping -n 2 -l 10 127.0.0.1 >nul
GoTo top
